towhee/tests/operators/test_pytorch_transform_operator.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+from PIL import Image
+from torchvision import transforms
+from torchvision import models
+import os
+
+from towhee.tests.mock_operators import PYTORCH_TRANSFORME_OPERATOR_PATH, load_local_operator
+
+
+class PytorchTransformTest(unittest.TestCase):
+    pwd = os.getcwd()
+    print('pwd is ' + str(pwd))
```

Comment:
We can leave these `print`s here for now, but let's get rid of them later on.

Reply:
ok